functions.php

```diff
@@ -134,7 +134,7 @@ function popper_widgets_init() {
  */
 function popper_scripts() {
 
-	wp_register_style( 'popper-style', get_stylesheet_uri() );
+	wp_register_style( 'popper-style', get_template_directory_uri() . '/style.css', array(), '20161208' );
```

Comment:
Later, liek WP 4.7 is in the wild active and the Theme can usage this version as minimum, we should switch to `get_theme_file_uri()`.